go.mod

```diff
@@ -97,11 +97,11 @@ require (
 	github.com/xsleonard/go-merkle v1.1.0
 	go.uber.org/mock v0.4.0
 	go.uber.org/zap v1.27.0
-	golang.org/x/crypto v0.22.0
```

Comment:
should we use same version of these libraries as op-geth?

Reply:
It may not the critical change for Granite. I think we should follow upstream erigon other than op-geth for library versions